test/persistence-hooks.suite.js

```diff
@@ -402,6 +449,112 @@ module.exports = function(dataSource, should) {
           });
       });
 
+      if (dataSource.connector.findOrCreate) {
+        it('triggers `persist` hook on findOrCreate when found', function(done) {
```

Comment:
Ditto, please remove " on findOrCreate".

I'll stop commenting on this, could you please fix names of all newly added tests?